src/spectral.ts

```diff
@@ -11,11 +13,28 @@ import {
   isOpenApiv3,
 } from '@stoplight/spectral';
 
+import * as SpectralPackage from '../node_modules/@stoplight/spectral/package.json';
 import { tryCatch } from 'fp-ts/lib/TaskEither';
 import { toError } from 'fp-ts/lib/Either';
+import { info } from '@actions/core';
+import { pluralizer } from './utils';
 
-export const createSpectral = (ruleset: string) => {
+const evaluateNumberOfExceptions = (exceptions: Record<string, string[]>) => {
+  const set = new Set();
+  let howMany = 0;
+
+  for (const path of Object.keys(exceptions)) {
+    set.add(path.split('#')[0]);
+    howMany += exceptions[path].length;
+  }
```

Comment:
You can probably rewrite this to make it not mutating the state and and reduce/lift the final count

Reply:
@XVincentX Fixed.